src/main/kotlin/app/revanced/patches/youtube/misc/bottomsheet/hook/BottomSheetHookResourcePatch.kt

```diff
@@ -0,0 +1,22 @@
+package app.revanced.patches.youtube.misc.bottomsheet.hook
+
+import app.revanced.patcher.data.ResourceContext
+import app.revanced.patcher.patch.PatchException
+import app.revanced.patcher.patch.ResourcePatch
+import app.revanced.patcher.patch.annotation.Patch
+import app.revanced.patches.shared.mapping.misc.ResourceMappingPatch
+
+@Patch(
+    name = "Remove root detection",
+    description = "Removes the check for root permissions.",
+    dependencies = [ResourceMappingPatch::class]
+)
```

Comment:
❓ : Mismatch patch configuration
> Name and description doesn't seems to be belong in `BottomSheetHookResourcePatch.kt`

```suggestion
@Patch(dependencies = [ResourceMappingPatch::class])
```